content/en/docs/contribute/new-content/new-features.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Documenting a new feature
+linktitle: New features
+content_template: templates/concept
+main_menu: true
+weight: 20
+card:
+  name: contribute
+  weight: 45
+  title: Documenting a new feature
+---
+{{% capture overview %}}
+
+Each major Kubernetes release includes new features.
+Most of these features need documentation to show people how to use them.
+
+Generally, the SIG responsible for a feature submits draft documentation for the
+feature as a pull request to the appropriate release branch of
+`kubernetes/website` repository, and someone on the SIG Docs team provides
+editorial feedback or edits the draft directly. This section covers the branching
+conventions and process used by both groups.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## For documentation contributors
+
+As stated above, usually the SIG creating a new feature submits draft documentation.
+This means that your role may be
+more of a shepherding role for a given feature than developing the documentation
+from scratch.
```

Comment:
Why is this paragraph/section necessary?

Reply:
It's wordy, but I do think it's needed – it introduces the idea that as a 'pure' docs contributor you might not be doing much writing. I'll take a stab at rewording it.